packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -29,8 +29,12 @@ class SemanticsService {
   ///
   /// For example a camera application can use this method to make accessibility
   /// announcements regarding objects in the viewfinder.
-  static Future<void> announce(String message, TextDirection textDirection) async {
-    final AnnounceSemanticsEvent event = AnnounceSemanticsEvent(message, textDirection);
+  ///
+  /// The assertiveness level of the announcement can be determined by [assertiveness].
+  /// Currently, this is only supported by the web engine and has no effect on
+  /// other platforms. The default mode is 'polite'.
```

Comment:
nit: when referring to API symbols, prefer backticks instead of single quotes, so it appears as `polite` rather than 'polite'.

Reply:
Or even better, link right to the value with `[Assertiveness.polite]`.